vernac/vernacentries.ml

```diff
@@ -2208,7 +2208,7 @@ let with_fail st b f =
       | HasNotFailed ->
           user_err ~hdr:"Fail" (str "The command has not failed!")
       | HasFailed msg ->
-          if not !Flags.quiet || !Flags.test_mode || !Flags.ide_slave then Feedback.msg_info
+          if not !Flags.quiet || !Flags.test_mode then Feedback.msg_info
```

Comment:
Did you test if coqide still prints something when you execute a `Fail Check 3 3.` ?

Reply:
IIRC the "verbose" flag is kind of randomly set...

Reply:
Yeah the `verbose` flag... I guess if such display is wanted in CoqIDE then it should set the flag itself, right?

I don't get this special case.

Reply:
In all cases `Flags.quiet` is used negatively so IMHO it should be renamed to `-verbose`. Similarly for `if_verbose` vs `if_silent`. I think a bit of cleanup there would be possible and great to do, but we need to wait for some PRs to be merged as they are fixing some other stuff [for example `echo` vs `verbose` overloading].

Reply:
I've no clue, I just asked if you tried.

Reply:
Yup, `coqide` by default runs with `quiet` on, so unless you do `Unset Silent` `Fail` will print nothing now.